angular-ui/src/app/search/search.component.html

```diff
@@ -2,22 +2,38 @@
   <div class="row">
     <div class="col-sm-12">
       <br/>
+      <a routerLink=""><button type="button" class="btn btn-dark">HOME</button></a><br><br>
       <ngb-alert type="secondary" [dismissible]="false">
         Currently, you can only search for cards by name. There are plans to expand search functionality to allow for more
         complex queries, such as searching by validity, card tags, card/creature type, color, and CMC.
       </ngb-alert>
       <div id="imaginary_container">
+      <h2>Search by card name:</h2>
         <div class="input-group stylish-input-group">
+
           <input [(ngModel)]="cardName" type="text" class="form-control"  placeholder="Search" >
           <span class="input-group-addon">
-                        <button (click)="onSearch(cardName)" type="submit" aria-label="submit">
+            <button (click)="onSearch(cardName)" type="submit" aria-label="submit">
+                <i class="fa fa-search" aria-hidden="true"></i>
+            </button>
+          </span>
+        </div>
+
+<!--         <div id="imaginary_container">
+        <h2>Search by card color:</h2>
+        <div class="input-group stylish-input-group">
+          <input [(ngModel)]="cardColor" type="text" class="form-control"  placeholder="Search" >
+          <span class="input-group-addon">
+                        <button (click)="onSearch(cardColor)" type="submit" aria-label="submit">
                             <i class="fa fa-search" aria-hidden="true"></i>
                         </button>
                     </span>
         </div>
-      </div><br>
-      <a routerLink=""><button type="button" class="btn btn-dark">HOME</button></a>
+ -->      </div><br>
+      
     </div>
+    <div *ngIf="card.tags"><h1>Sorted</h1></div>

```

Comment:
Can you make the text here match the style of the rest of the app? (font-family, font-size, text-align: center, etc)